WORKSPACE

```diff
@@ -39,9 +39,10 @@ llvm_toolchain(
     name = "llvm_toolchain_15_0_7",
     absolute_paths = True,
     llvm_mirror_prefixes = [
-        "https://sorbet-deps.s3-us-west-2.amazonaws.com/",
         "https://artifactory-content.stripe.build/artifactory/github-archives/llvm/llvm-project/releases/download/llvmorg-",
         "https://github.com/llvm/llvm-project/releases/download/llvmorg-",
+        "https://artifactory-content.stripe.build/artifactory/github-archives/llvm/llvm-project/releases/download/llvmorg-",
```

Comment:
We have artifactory-content.stripe.build in here twice?  Is that intended?

Reply:
Good catch, I guess that maybe Stripe's CI automatically rewrites github.com URLs to artifactory URLs? I wonder if that means we can drop all of our hacks to attempt to fetch from artifactory here. I will flag with our Build team.

Reply:
From Stripe's build team: we shouldn't need the artifactory URLs at all.

This branch's CI is passing and approved, and I'd kind of like to land this change and bump Sorbet in Stripe's codebase, so I'm going to take a follow up change to tinker with the artifactory URLs.

It will be nice to remove those URLs because they print bazel WARNING lines when those hosts fail to resolve.